app/domain/operations/migrations/people/migrate_ai_an_to_attested_status.rb

```diff
@@ -0,0 +1,173 @@
+        #
+        # @param [Person] person: The person to process.
+        # @param [VerificationType] verification_type: The verification type to process.
+        # @param [Symbol] mode: The operation mode.
+        def data_row(person, verification_type, mode)
+          mode == :data_fix ? data_fix_row(person, verification_type) : default_row(person, verification_type)
+        end
+
+        def data_fix_row(person, verification_type)
+          [
+            person.hbx_id,
+            verification_type.type_name,
+            verification_type.inactive,
+            verification_type.validation_status,
+            verification_type.update_reason,
+            verification_type.due_date,
+            'attested',
+            'Policy change to accept self-attestation of American Indian status',
+            nil,
+            'Migrate to attested',
+            'System',
+            'Policy change to accept self-attestation of American Indian status',
+            nil,
+            nil,
+            verification_type.validation_status,
+            'attested'
+          ]
+        end
+
+        def default_row(person, verification_type)
+          last_type_history_element = verification_type.type_history_elements&.last
+
+          [
+            person.hbx_id,
+            verification_type.type_name,
+            verification_type.inactive,
+            verification_type.validation_status,
+            verification_type.update_reason,
+            verification_type.due_date,
+            last_type_history_element&.action,
+            last_type_history_element&.modifier,
+            last_type_history_element&.update_reason,
+            last_type_history_element&.event_response_record_id,
+            last_type_history_element&.event_request_record_id,
+            last_type_history_element&.from_validation_status,
+            last_type_history_element&.to_validation_status
+          ]
+        end
+
+        # Updates impacted cases, records fixes, or exports impacted case data.
+        #
+        # @param [Array<Person>] people: The list of people to process.
+        # @param [Symbol] mode The operation mode: either `:data_fix` or `:report`.
+        # @return [Dry::Monads::Result::Success, Dry::Monads::Result::Failure]
+        # Success with the result of processing, or Failure with an error message if an error occurs.
+        def migrate(people, mode)
+          initialize_csv(mode)
+          people.each do |person|
+
+            american_indian_status = person.american_indian_status
+            next unless american_indian_status
+            next unless VerificationType::NEGATIVE_STATES.include?(american_indian_status.validation_status)
+
+            data = data_row(person, american_indian_status, mode)
+            append_csv_row(data, mode)
+
+            next unless mode == :data_fix
+            update_person(american_indian_status)
```

Comment:
Realize this makes things a bit more complicated, but I think we should do the person update before populating the complete row when in `data_fix` mode.  Then we can populate the remaining row data using the actual updated values on the record.  Reason being that we should not assume the record will update successfully.  If it fails to update for some totally unrelated reason, we want to log the error in the report for that record.